xbmc/cores/AudioEngine/Sinks/AESinkDirectSound.cpp

```diff
@@ -218,10 +218,10 @@ bool CAESinkDirectSound::Initialize(AEAudioFormat &format, std::string &device)
 
   m_AvgBytesPerSec = wfxex.Format.nAvgBytesPerSec;
 
-  unsigned int uiFrameCount = (int)(format.m_sampleRate * 0.015); //default to 15ms chunks
+  const unsigned int uiFrameCount = static_cast<int>(format.m_sampleRate * 0.050); // 50ms chunks
   m_dwFrameSize = wfxex.Format.nBlockAlign;
   m_dwChunkSize = m_dwFrameSize * uiFrameCount;
-  m_dwBufferLen = m_dwChunkSize * 12; //180ms total buffer
+  m_dwBufferLen = m_dwChunkSize * 8; // 400ms total buffer
```

Comment:
I have a similar issue in pulse and Linux. It might very well be that just the increased chunk size is enough.

Reply:
https://github.com/xbmc/xbmc/blob/5e6d5de2c6a151683b77ac34abce8fae8294b7b9/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp#L733

Reply:
> https://github.com/xbmc/xbmc/blob/5e6d5de2c6a151683b77ac34abce8fae8294b7b9/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp#L733

https://github.com/xbmc/xbmc/blob/5e6d5de2c6a151683b77ac34abce8fae8294b7b9/xbmc/cores/AudioEngine/Sinks/AESinkPULSE.cpp#L907

Reply:
Your change is fine. Let's assume Get delay works properly combined with Flush.